cni/network/invoker.go

```diff
@@ -31,6 +31,15 @@ type IPAMAddResult struct {
 	ipv6Enabled bool
 }
 
+func (ipamAddConfig IPAMAddConfig) getOptions() *map[string]interface{} {
```

Comment:
since this is a function that is a part of IPAMAddConfig can we rename this to like `getShallowCopy` or `shallowCopy` or something like that

Reply:
also I think map[string]interface{} is already a reference to the underlying map so you don't need the * indicating a pointer. The value of res itself can be thought of as an address (not the actual data).